### PR TITLE
zh-cn(fix): fix error markdown syntax

### PR DESCRIPTION
**Pull request by daiwanxing**
### Description

found an error markdown bold syntax when read [queueMicroTask Chapter](https://developer.mozilla.org/zh-CN/docs/Web/API/HTML_DOM_API/Microtask_guide)

### CODE REVIEW

File: files/zh-cn/web/api/html_dom_api/microtask_guide/index.md [hunk 1]
```diff
@@ -5,9 +5,9 @@ slug: Web/API/HTML_DOM_API/Microtask_guide
 
 {{APIRef("HTML DOM")}}
 
-一个 **微任务（microtask）**就是一个简短的函数，当创建该函数的函数执行之后，_并且_ 只有当 Javascript 调用栈为空，而控制权尚未返还给被 {{Glossary("user agent")}} 用来驱动脚本执行环境的事件循环之前，该微任务才会被执行。事件循环既可能是浏览器的主事件循环也可能是被一个 [web worker](/zh-CN/docs/Web/API/Web_Workers_API) 所驱动的事件循环。这使得给定的函数在没有其他脚本执行干扰的情况下运行，也保证了微任务能在用户代理有机会对该微任务带来的行为做出反应之前运行。
+一个 **微任务(microtask)** 就是一个简短的函数，当创建该函数的函数执行之后，*并且* 只有当 Javascript 调用栈为空，而控制权尚未返还给被 {{Glossary("user agent")}} 用来驱动脚本执行环境的事件循环之前，该微任务才会被执行。事件循环既可能是浏览器的主事件循环也可能是被一个 [web worker](/zh-CN/docs/Web/API/Web_Workers_API) 所驱动的事件循环。这使得给定的函数在没有其他脚本执行干扰的情况下运行，也保证了微任务能在用户代理有机会对该微任务带来的行为做出反应之前运行。
```
Comment (yin1999) [inline files/zh-cn/web/api/html_dom_api/microtask_guide/index.md:8]:
```suggestion
一个**微任务**（microtask）就是一个简短的函数，当创建该函数的函数执行之后，*并且*只有当 Javascript 调用栈为空，而控制权尚未返还给被{{Glossary("user agent", "用户代理")}}用来驱动脚本执行环境的事件循环之前，该微任务才会被执行。事件循环既可能是浏览器的主事件循环也可能是被一个 [web worker](/zh-CN/docs/Web/API/Web_Workers_API) 所驱动的事件循环。这使得给定的函数在没有其他脚本执行干扰的情况下运行，也保证了微任务能在用户代理有机会对该微任务带来的行为做出反应之前运行。
```

File: files/zh-cn/web/api/html_dom_api/microtask_guide/index.md [hunk 2]
```diff
@@ -5,9 +5,9 @@ slug: Web/API/HTML_DOM_API/Microtask_guide
...
 
-JavaScript 中的 [promises](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise) 和 [Mutation Observer API](/zh-CN/docs/Web/API/Mutation_Observer_API) 都使用微任务队列去运行它们的回调函数，但当能够推迟工作直到当前事件循环过程完结时，也是可以执行微任务的时机。为了允许第三方库、框架、polyfills 能使用微任务，{{domxref("Window")}} 暴露了 {{domxref("queueMicrotask()")}} 方法，而 {{domxref("Worker")}} 接口则通过 `WindowOrWorkerGlobalScope` mixin 提供了同名的 queueMicrotask() 方法。
+JavaScript 中的 [promises](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise) 和 [Mutation Observer API](/zh-CN/docs/Web/API/MutationObserver) 都使用微任务队列去运行它们的回调函数，但当能够推迟工作直到当前事件循环过程完结时，也是可以执行微任务的时机。为了允许第三方库、框架、polyfills 能使用微任务，在{{domxref("Window")}} 和 {{domxref("Worker")}} 接口上暴露了{{domxref("queueMicrotask()")}}方法。
```
Comment (yin1999) [inline files/zh-cn/web/api/html_dom_api/microtask_guide/index.md:10]:
```suggestion
JavaScript 中的 [promise](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise) 和 [Mutation Observer API](/zh-CN/docs/Web/API/MutationObserver) 都使用微任务队列去运行它们的回调函数，但当能够推迟工作直到当前事件循环过程完结时，也是可以执行微任务的时机。为了允许第三方库、框架、polyfill 能使用微任务，在 {{domxref("Window")}} 和 {{domxref("Worker")}} 接口上暴露了{{domxref("queueMicrotask()")}}方法。
```

**Review by yin1999 — COMMENTED** [review]
Thanks @daiwanxing, We just follow some guidelines which is listed [here](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md).

I've followed this, and list some suggestions here.

File: files/zh-cn/web/api/html_dom_api/microtask_guide/index.md (hunk 1 above)
Comment (daiwanxing) [reply files/zh-cn/web/api/html_dom_api/microtask_guide/index.md:8]: Thanks  your suggets. but in here `一个**微任务**（microtask）就是一个简短的函数`, if not append spaces after the right flanking, the bold text will not effecct.

File: files/zh-cn/web/api/html_dom_api/microtask_guide/index.md (hunk 2 above)
Comment (daiwanxing) [reply files/zh-cn/web/api/html_dom_api/microtask_guide/index.md:10]: OK, I'll modify it.

File: files/zh-cn/web/api/html_dom_api/microtask_guide/index.md (hunk 1 above)
Comment (yin1999) [reply files/zh-cn/web/api/html_dom_api/microtask_guide/index.md:8]: You can try my suggestion, this fix would be ok.

Comment (daiwanxing) [reply files/zh-cn/web/api/html_dom_api/microtask_guide/index.md]: ok,now it's fixed.

File: files/zh-cn/web/api/html_dom_api/microtask_guide/index.md [hunk 3]
```diff
@@ -5,9 +5,10 @@ slug: Web/API/HTML_DOM_API/Microtask_guide
 
 {{APIRef("HTML DOM")}}
 
-一个 **微任务（microtask）**就是一个简短的函数，当创建该函数的函数执行之后，_并且_ 只有当 Javascript 调用栈为空，而控制权尚未返还给被 {{Glossary("user agent")}} 用来驱动脚本执行环境的事件循环之前，该微任务才会被执行。事件循环既可能是浏览器的主事件循环也可能是被一个 [web worker](/zh-CN/docs/Web/API/Web_Workers_API) 所驱动的事件循环。这使得给定的函数在没有其他脚本执行干扰的情况下运行，也保证了微任务能在用户代理有机会对该微任务带来的行为做出反应之前运行。
```
Comment (yin1999) [inline files/zh-cn/web/api/html_dom_api/microtask_guide/index.md:8]:
```suggestion
```

File: files/zh-cn/web/api/html_dom_api/microtask_guide/index.md [hunk 4]
```diff
@@ -5,9 +5,9 @@ slug: Web/API/HTML_DOM_API/Microtask_guide
...
+一个**微任务**（microtask）就是一个简短的函数，当创建该函数的函数执行之后，*并且* 只有当 Javascript 调用栈为空，而控制权尚未返还给被 {{Glossary("user agent", "用户代理")}} 用来驱动脚本执行环境的事件循环之前，该微任务才会被执行。事件循环既可能是浏览器的主事件循环也可能是被一个 [web worker](/zh-CN/docs/Web/API/Web_Workers_API) 所驱动的事件循环。这使得给定的函数在没有其他脚本执行干扰的情况下运行，也保证了微任务能在用户代理有机会对该微任务带来的行为做出反应之前运行。
```
Comment (yin1999) [inline files/zh-cn/web/api/html_dom_api/microtask_guide/index.md:8]:
```suggestion
一个**微任务**（microtask）就是一个简短的函数，当创建该函数的函数执行之后，*并且*只有当 Javascript 调用栈为空，而控制权尚未返还给被 {{Glossary("user agent", "用户代理")}} 用来驱动脚本执行环境的事件循环之前，该微任务才会被执行。事件循环既可能是浏览器的主事件循环也可能是被一个 [web worker](/zh-CN/docs/Web/API/Web_Workers_API) 所驱动的事件循环。这使得给定的函数在没有其他脚本执行干扰的情况下运行，也保证了微任务能在用户代理有机会对该微任务带来的行为做出反应之前运行。
```

**Review by yin1999 — APPROVED** [review]
Thanks @daiwanxing. Welcome aboard :)

File: files/zh-cn/web/api/html_dom_api/microtask_guide/index.md [hunk 5]
```diff
@@ -5,9 +5,9 @@ slug: Web/API/HTML_DOM_API/Microtask_guide
...
+一个**微任务**（microtask）就是一个简短的函数，当创建该函数的函数执行之后，*并且*只有当 Javascript 调用栈为空，而控制权尚未返还给被{{Glossary("user agent", "用户代理")}}用来驱动脚本执行环境的事件循环之前，该微任务才会被执行。事件循环既可能是浏览器的主事件循环也可能是被一个 [web worker](/zh-CN/docs/Web/API/Web_Workers_API) 所驱动的事件循环。这使得给定的函数在没有其他脚本执行干扰的情况下运行，也保证了微任务能在用户代理有机会对该微任务带来的行为做出反应之前运行。
 
-JavaScript 中的 [promises](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise) 和 [Mutation Observer API](/zh-CN/docs/Web/API/Mutation_Observer_API) 都使用微任务队列去运行它们的回调函数，但当能够推迟工作直到当前事件循环过程完结时，也是可以执行微任务的时机。为了允许第三方库、框架、polyfills 能使用微任务，{{domxref("Window")}} 暴露了 {{domxref("queueMicrotask()")}} 方法，而 {{domxref("Worker")}} 接口则通过 `WindowOrWorkerGlobalScope` mixin 提供了同名的 queueMicrotask() 方法。
+JavaScript 中的 [promise](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise) 和 [Mutation Observer API](/zh-CN/docs/Web/API/MutationObserver) 都使用微任务队列去运行它们的回调函数，但当能够推迟工作直到当前事件循环过程完结时，也是可以执行微任务的时机。为了允许第三方库、框架、polyfill 能使用微任务，在 {{domxref("Window")}} 和 {{domxref("Worker")}} 接口上暴露了{{domxref("queueMicrotask()")}}方法。
```
Comment (yin1999) [inline files/zh-cn/web/api/html_dom_api/microtask_guide/index.md:10]:
```suggestion
JavaScript 中的 [promise](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Promise) 和 [Mutation Observer API](/zh-CN/docs/Web/API/MutationObserver) 都使用微任务队列去运行它们的回调函数，但当能够推迟工作直到当前事件循环过程完结时，也是可以执行微任务的时机。为了允许第三方库、框架、polyfill 能使用微任务，在 {{domxref("Window")}} 和 {{domxref("Worker")}} 接口上暴露了 {{domxref("queueMicrotask()")}} 方法。
```

**Pull request merged**